test/common/upstream/health_check_fuzz.proto

```diff
@@ -13,8 +13,45 @@ message HttpRespond { //TODO: Split this across headers, body, and trailers?
   string status = 2 [(validate.rules).string = {pattern: "[0-9]+$*", min_len: 1}];
 }
 
+message TcpRespond {
+  bytes data = 1;
+}
+
+message GrpcRespondHeaders {
+  test.fuzz.Headers headers = 1;
+  //https://github.com/nodejs/http-parser/blob/master/http_parser.c#L852
+  string status = 2 [(validate.rules).string = {pattern: "[0-9]+$*", min_len: 1}];
```

Comment:
same comment from HTTP? maybe this should be a uint64_t now

Reply:
Yup, will switch.